### PR TITLE
mnt: update pytorch versions

### LINKED ISSUE #2691: investigate pytorch dependency issues (with cuda)
seems that recent changes to pytorch (on pytorch channel) seem to lead to some insolvable envs (at least on windows).

### PR DESCRIPTION
**Pull request by k-dominik**
fixes #2691

### CODE REVIEW

File: conda-recipe/meta.yaml [hunk 1]
```diff
@@ -120,7 +120,7 @@ outputs:
         - pytest >=3,<4
         - pytest-qt
         # need to help mamba here a bit
```
Comment (btbest) [inline conda-recipe/meta.yaml:122]: I guess this comment can go now?

**Review by btbest — APPROVED** [review]

File: conda-recipe/meta.yaml (hunk 1 above)
Comment (k-dominik) [reply conda-recipe/meta.yaml:122]: ah crap I moved stuff around - this should actually stay, but need to move down the `ilastik-pytorch-version-helper-cpu` thing...

Comment (k-dominik) [reply conda-recipe/meta.yaml:122]: or the comment up :)

Comment (k-dominik) [reply conda-recipe/meta.yaml:122]: ah, nah, this should actually be in test reqs - good catch thx :)

**Pull request merged**